pom.xml

```diff
@@ -70,10 +70,16 @@ THE SOFTWARE.
 
   <repositories>
     <repository>
-      <id>mavenTMateSoftReleasesRepository</id>
-      <url>http://maven.tmatesoft.com/content/repositories/releases/</url>
+      <id>repo.jenkins-ci.org</id>
+      <url>https://repo.jenkins-ci.org/public/</url>
     </repository>
   </repositories>
+  <pluginRepositories>
+    <pluginRepository>
+      <id>repo.jenkins-ci.org</id>
+      <url>https://repo.jenkins-ci.org/public/</url>
+    </pluginRepository>
+  </pluginRepositories>
```

Comment:
It is, but before the parent POM can even be applied, the `packaging` must be resolved…and `maven-hpi-plugin` is not in Central. Therefore if you remove this `pluginRepository`, the build will fail in a clean environment. (Of course the `repository` is also needed, else the parent POM cannot even be loaded.)